src/bus/policy.c

```diff
@@ -1082,6 +1114,16 @@ int policy_snapshot_check_receive(PolicySnapshot *snapshot,
                                   size_t n_fds) {
         PolicyVerdict verdict = POLICY_VERDICT_INIT;
         size_t i;
+        int r;
+
+        r = bus_apparmor_check_xmit(snapshot->apparmor, false, sender_seclabel, receiver_seclabel,
```

Comment:
This is now: `subject_seclabel, snapshot->seclabel`, rather than `sender_seclabel, receiver_seclabel`